artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/components/addressSendMessage.js

```diff
@@ -43,6 +43,10 @@ var Artemis;
                             <label>Durable </label>
                             <input id="durable" type="checkbox" ng-model="$ctrl.message.durable" value="true">
                         </div>
+                        <div class="form-group">
+                            <label>Create User ID </label>
```

Comment:
I think, call this message id, it does not identify the user, it is a user supplied id for the message.

Reply:
It isnt really a user supplied id in this case, or any sent with a JMS client, as its being generated by something else. YEt more reason to reference the message id it is.